airflow-core/src/airflow/ui/src/i18n/locales/pl/assets.json

```diff
@@ -0,0 +1,29 @@
+{
+  "consumingDags": "Pobieranie DAG-ów",
+  "createEvent": {
+    "button": "Utwórz zdarzenie",
+    "manual": {
+      "description": "Utwórz bezpośrednio zdarzenie zasobu",
+      "extra": "Dodatkowe informacje o zdarzeniu zasobu",
+      "label": "Ręcznie"
+    },
+    "materialize": {
+      "description": "Wyzwól DAG zależny od tego zasobu",
+      "descriptionWithDag": "Wyzwól DAG zależny od tego zasobu: {{dagName}}",
+      "label": "Materializuj",
+      "unpauseDag": "Wznów {{dagName}} przy wyzwoleniu"
+    },
+    "success": {
+      "manualDescription": "Ręczne utworzenie zdarzenia zasobu zakończone sukcesem.",
+      "manualTitle": "Zdarzenie zasobu utworzone",
+      "materializeDescription": "Zależny DAG {{dagId}} został pomyślnie wyzwolony.",
```

Comment:
Maybe it should be `wywołany` than `wyzwolony`? I think more of a "liberated" than "triggered" when hearing "wyzwolony".

Reply:
Yep.